SU2_CFD/include/variables/CNEMOEulerVariable.hpp

```diff
@@ -27,48 +27,34 @@
 
 #pragma once
 
-#include "CVariable.hpp"
+#include "CFlowVariable.hpp"
 #include "../fluid/CNEMOGas.hpp"
 #include "../../../Common/include/toolboxes/geometry_toolbox.hpp"
 
 /*!
  * \class CNEMOEulerVariable
  * \brief Main class for defining the variables of the NEMO Euler's solver.
+ * \note Primitive variables (rhos_s, T, Tve, ...)
```

Comment:
Can somone from the NEMO-team add the `...` the the `Gradients of Primitives` part like in the other solvers @WallyMaier @CatarinaGarbacz , or is this not really applicable here?

Reply:
@TobiKattmann I can fill out the primitives.  I need to double check how much of the Gradient/Limiter primitive is filled out.